client-spark/shuffle-manager-2/pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.celeborn</groupId>
-        <artifactId>remote-shuffle-service</artifactId>
+        <artifactId>celeborn-parent</artifactId>
```

Comment:
idealy, we should add _2.12 suffix, but it makes the compile complex because
1) as we support spark 2.4 ~ 3.3, then we should support scala 2.11 ~ 2.13
2) maven force to use literal instead property in parent `artifactId`

so we should introduce `dev/change-scala-version.sh` like Spark does.